src/main/scala/com/microsoft/hyperspace/util/PathUtils.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.util
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+
+object PathUtils {
+
+  def makeAbsolute(location: String): Path = makeAbsolute(new Path(location))
+
+  def makeAbsolute(path: Path): Path = {
+    val fs = path.getFileSystem(new Configuration)
+    fs.makeQualified(path)
```

Comment:
I guess already absolute path is a no-op here, right?

Reply:
yes that's correct. I could add the check (if not absolute then update) but I thought this looks cleaner